### ISSUE TITLE
Support extended Newick?

### ISSUE THREAD

**Issue by cossio** [arzwa/NewickTree.jl#4]
It seems the package cannot read extended Newick format trees?

I'm having trouble in particular reading trees stored in RFAM (https://docs.rfam.org/en/latest/api.html?highlight=nhx#tree-data). For example,

http://rfam.org/family/RF00360/tree/

**Comment by arzwa** [comment]
No, currently this package only supports the Newick format as defined by Joe Felsenstein here (https://evolution.genetics.washington.edu/phylip/newicktree.html), at least I hope it does so :). Due to a shift of focus, I am not actively using/developing this package right now, so I won't work on this, but I  would welcome PRs with extensions of course.
